python/cli/prelude_cli/views/iam.py

```diff
@@ -201,6 +201,16 @@ def unsubscribe(controller, event):
         data = controller.unsubscribe(event=AuditEvent[event])
     print_json(data=data)
 
+iam.command('accept-terms')
+@click.argument('name', type=str, required=True)
+@click.option('-v', '--version', type=int, required=True)
+@click.pass_obj
+@handle_api_error
+def accept_terms(controller, name, version):
+    """ Accept terms and conditions """
+    with Spinner(description='Accepting terms and conditions'):
+        data = controller.accept_terms(name=name, version=version)
+    print_json(data=data)
```

Comment:
any legal issues with accepting terms and conditions we haven't given them?

Reply:
idk.. the http endpoint is there and open whether the CLI cmd exists or not. I didn't add a `choice` to the `name` argument so it'll still be somewhat difficult to know what the valid choices are. also making the cmd hidden so it won't show up for users

do you think there's legal issue with the sdk function existing?

Reply:
ok that sounds reasonable to me. Proceed!